group/templates/group/groupTable.html

```diff
@@ -0,0 +1,56 @@
+{% load group_extras %}
+<table class="table table-hover">
+  <thead>
+    <tr>
+      <th width="15%">
```

Comment:
I suggest not fixing width=15% here.
If a user doesn't have edit auth, it will be a little wired.